### ISSUE TITLE
Material type is confusing

### ISSUE THREAD

**Issue by mrmanc** [ashwanthkumar/gocd-build-github-pull-requests#146]
Is there a reason that the material type is `Github` rather than `GitHub Pull Requests`? I think people have missed it as an option because the difference between `Git` and `Github` is so subtle.

**Comment by ashwanthkumar** [comment]
No specific reason. It's been that way since the day 1 as far as I can recall. Most of us are now used seeing `Github` material type. I'll keep this issue open for a while, if more users thing it should be renamed.

**Comment by kuba-lilz** [comment]
While I got used to it by now, I indeed found it very confusing when starting.

**Comment by stephen-murby** [comment]
I agree I think this would be a much less confusing material name, I will include a rename in my (work in progress) pull request to filter out draft pull requests from triggering a pipeline run.
